src/components/application_manager/src/hmi_state.cc

```diff
@@ -396,6 +408,10 @@ mobile_apis::HMILevel::eType EmbeddedNavi::max_hmi_level() const {
   using namespace mobile_apis;
   using namespace helpers;
 
+  if (parent() && HMILevel::INVALID_ENUM == parent()->hmi_level()) {
+    return parent()->hmi_level();
+  }
```

Comment:
```suggestion
  if (HMILevel::INVALID_ENUM == parent_hmi_level()) {
    return parent_hmi_level();
  }
```
This function has a null check for `parent()` when necessary